### PR TITLE
Math logic PR

### PR DESCRIPTION
**Pull request by AmaduKamara**
# Calculator App - Logic implementation
Implemented the following requirements: ✔️
- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, be sure you understand what they do.
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift the state up.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,86 +1,212 @@
...
+                  >
+                    +/-
+                  </button>
+                </td>
+                <td className="bg-white border hover:bg-gray-200 text-center">
+                  <button
+                    className="p-6"
+                    type="button"
+                    onClick={this.handleClick}
+                  >
```
Comment (SajjadAhmad14) [inline src/components/Calculator.js:50]: Please create a child component of the `Calculator` component and pass `handleClick` as props to it.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Amkam, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Calculator.js (hunk 1 above)
Comment (AmaduKamara) [reply src/components/Calculator.js:50]: Hello Sajjad, creating a child component for the Calculator that is empty is kind of getting me, confused bro. Don't know exactly what that component would do. What do you think?

Comment (AmaduKamara) [reply src/components/Calculator.js:50]: I'm having a single Calculator component and all the logic is working as expected.

Comment (SajjadAhmad14) [reply src/components/Calculator.js:50]: Hi, You could make another component named `Button`, put all the logic of button there and pass `handleClick` event as props to that child component.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Amkam, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**